cmake/onnxruntime_util.cmake

```diff
@@ -10,14 +10,12 @@ file(GLOB_RECURSE onnxruntime_util_srcs CONFIGURE_DEPENDS
 
 source_group(TREE ${ONNXRUNTIME_ROOT}/core FILES ${onnxruntime_util_srcs})
 
-set(gemmlowp_src ${PROJECT_SOURCE_DIR}/external/gemmlowp)
```

Comment:
Please also remove gemmplowp from .gitmodules

Reply:
I'm going to remove the git module in a different PR